apps/tangle-dapp/hooks/useEvmAddress.ts

```diff
@@ -0,0 +1,31 @@
+import { useMemo } from 'react';
+
+import { isEvmAddress } from '../utils/isEvmAddress';
+import useActiveAccountAddress from './useActiveAccountAddress';
+
+export type EvmAddressOrHash = `0x${string}`;
```

Comment:
Already have a type for this in the lib folder called `AddressType`. Feel free to change the name in the lib but note that the `AddressType` type is used across many dapps we have